app/src/main/java/com/github/houseorganizer/houseorganizer/task/TaskView.java

```diff
@@ -21,16 +22,18 @@
 
 import com.google.firebase.firestore.DocumentReference;
 import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.QueryDocumentSnapshot;
 
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.function.Consumer;
 
 public final class TaskView {
```

Comment:
For this week no, it requires adding a task in my tests, and then deleting it. Unfortunately deleting tasks on firebase has a bug which will be fixed next sprint. After that, it will be trivial to raise the coverage here, sure.